src/Arduino_WiFiConnectionHandler.cpp

```diff
@@ -85,142 +101,197 @@ unsigned long WiFiConnectionHandler::getTime() {
 void WiFiConnectionHandler::update() {
 
   unsigned long const now = millis();
-  if((now - lastConnectionTickTime) > connectionTickTimeInterval)
-  {
+  int networkStatus = 0;
```

Comment:
You have deleted existing functionality here because probably your work has been based on a older version of `Arduino_ConnectionHandler`. Please restore.